system-int-test/src/cmr/system_int_test/data2/umm_spec_service.clj

```diff
@@ -0,0 +1,67 @@
+(ns cmr.system-int-test.data2.umm-spec-service
+  "Contains service data generators for example-based testing in system
+  integration tests."
+  (:require
+   [cmr.common.mime-types :as mime-types]
+   [cmr.umm-spec.models.umm-service-models :as umm-s]
+   [cmr.umm-spec.test.location-keywords-helper :as lkt]
+   [cmr.umm-spec.umm-spec-core :as umm-spec]))
+
+(def context (lkt/setup-context-for-test))
+
+(def ^:private sample-umm-service
+  {:Name "AIRX3STD"
+   :Type "OPeNDAP"
+   :Version "1.9"
+   :Description "AIRS Level-3 retrieval product created using AIRS IR, AMSU without HSB.",
+   :OnlineResource {:Linkage "https://acdisc.gesdisc.eosdis.nasa.gov/opendap/Aqua_AIRS_Level3/AIRX3STD.006/",
+                    :Name "OPeNDAP Service for AIRS Level-3 retrieval products",
+                    :Description "OPeNDAP Service"}})
+
+(defn- service
+  "Returns a UMM-S record from the given attribute map."
+  ([]
+   (service {}))
+  ([attribs]
+   (umm-s/map->UMM-S (merge sample-umm-service attribs)))
+  ([index attribs]
+   (umm-s/map->UMM-S
+    (merge sample-umm-service
+           {:Name (str "Name " index)}
+           attribs))))
+
+(defn- umm-s->concept
+  "Returns a concept map from a UMM service item or tombstone."
+  ([item]
```

Comment:
I think umm-json is the only supported format. It would be nice to get rid of any of these multiple arrities that we don't actually need.

Reply:
Done.